fix(prometheus-switch-3.0.0): Adding 3.0.0 prometheus changelog docs (#4185)
Kong/docs.konghq.com: app/_hub/kong-inc/prometheus/3.0.0.md

   if the Status API is not enabled. Note that this endpoint is unavailable
   when [RBAC](/gateway/latest/admin-api/rbac/reference/) is enabled on the
   Admin API (Prometheus does not support Key-Auth to pass the token).

---

## Changelog

* Metric names have been adjusted to add units to better standardize where possible
  * `http_status` -> `http_requests_total`
  * `latency` -> `kong_request_latency_ms`/`kong_upstream_latency_ms`/`kong_kong_latency_ms`
  * `kong_bandwidh` -> `kong_bandwidth_bytes`
  * `nginx_http_current_connections`/`nginx_stream_current_connections` -> `nginx_hconnections_total`
  * `http_consumer_status` removed.
* New Metric: `session_duration_ms` for monitoring stream connections.
* New Metric: `node_info`. Single gauge set to 1 that outputs the node's id and kong version.
* Latency has been split into 4 different metrics: `kong_latency_ms`, `upstream_latency_ms` and `request_latency_ms` (http) /`session_duration_ms` (stream). Buckets details below.
  * Kong Latency and Upstream Latency can operate at orders of magnitudes different. Separate out these buckets to reduce memory overhead.

> * `request_count` and `consumer_status` have been merged into just `http_requests_total`. If the `per_consumer` config is set false, the `consumer` label will be empty.  If the `per_consumer` config is true, it will be filled.

Reviewer: ```suggestion
* `request_count` and `consumer_status` were merged into `http_requests_total`. If the `per_consumer` config is set to false, the `consumer` label will be empty.  If the `per_consumer` config is true, the `consumer` label will be filled.
```

> * `http_requests_total` has a new label `source`. set to either `exit`, `error` or `service`. https://docs.konghq.com/gateway/latest/pdk/kong.response/#kongresponseget_source

Reviewer: ```suggestion
* `http_requests_total` has a new label [`source`](/gateway/latest/pdk/kong.response/#kongresponseget_source). It can be set to `exit`, `error`, or `service`.
```

> * All Memory metrics have a new label `node_id`

Reviewer: ```suggestion
* All Memory metrics have a new label, `node_id`.
```

* `nginx_http_current_connections` merged with `nginx_stream_current_connection` to `nginx_current_connections`

> * Plugin Version Bumped to 3.0

Reviewer: ```suggestion
* Plugin version bumped to 3.0.x
```

> * The `node_id` label has been added to memory metrics.

Reviewer: ```suggestion
* The `node_id` label was added to memory metrics.
```
